django/db/models/fields/reverse_related.py

```diff
@@ -176,9 +177,6 @@ def get_accessor_name(self, model=None):
             return self.related_name
         return opts.model_name + ('_set' if self.multiple else '')
 
-    def get_cache_name(self):
```

Comment:
How does it work to just remove this method? The default implementation in the mixin returns `self.name`, but `ForeignObjectRel` doesn't appear to have a `name` attribute. Or was this method never actually used?